.github/workflows/pull-request.yml

```diff
@@ -23,15 +19,18 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@v4
+      - id: env 
+        name: Setup Dependencies Env Variables
+        uses: ./.github/actions/setup-build-env
       - name: Install dependencies
         env: 
-          HELM_URL_V3: https://get.helm.sh/helm-${{ env.HELM_VERSION_V3 }}-linux-amd64.tar.gz
+          HELM_URL: https://get.helm.sh/helm-${{ steps.env.outputs.HELM_VERSION }}-linux-amd64.tar.gz
         run: |
           sudo snap install yq
-          curl ${{ env.HELM_URL_V3 }} | tar xvzf - --strip-components=1 -C /tmp/ && \
+          curl ${{ env.HELM_URL }} | tar xvzf - --strip-components=1 -C /tmp/ && \
           sudo mv /tmp/helm /usr/bin/helm_v3 && \
```

Comment:
Do we still want to keep the binary as `helm_v3`?

Reply:
I know it is `helm_v3` for compatibility reasons, but I don't think there's a reason to not update it to just `helm` in the future.

Reply:
SGTM

Reply:
The only place I know we call this binary at runtime is this one: https://github.com/rancher/rancher/blob/3222eb4237c45065b16bf8e673a56794ab346741/package/jailer.sh#L64

If we were to change this, we are probably better off doing it for `v2.10` to avoid behaviour change on a patch release.